bitnami/postgresql-ha/CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # Changelog
 
-## 14.2.4 (2024-06-11)
+## 14.2.5 (2024-06-11)
 
-* [bitnami/postgresql-ha] Release 14.2.4 ([#27094](https://github.com/bitnami/charts/pull/27094))
+* Draft: [bitnami/postgresql-ha] Fix postgresql repmgr  generation ([#27095](https://github.com/bitnami/charts/pull/27095))
```

Comment:
This must be updated (contains the "draf" reference)

Reply:
done at https://github.com/bitnami/charts/pull/27095/commits/0b81e5abbbc67436a019b883c8945fd5ea4f6515